docs/openj9_defaults.md

```diff
@@ -81,7 +81,7 @@ The last 2 columns show whether the default setting can be changed by a command-
 |Maximum stack size for Java Threads **(32-bit)**. Use `-Xss<size>`  |320 KB    |320 KB    |N/A              |320 KB           |320 KB    |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
 |Maximum stack size for Java Threads **(64-bit)**. Use `-Xss<size>`  |1024 KB   |1024 KB   |1024 KB          |1024 KB          |1024 KB   |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
 |Stack size for OS Threads **(32-bit)**. Use `-Xmso<size>`           |256 KB    |256 KB    |N/A              |32 KB            |256 KB    |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
-|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`           |512 KB    |256 KB (512 KB on PPC)    |256 KB           |256 KB           |1 MB      |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
+|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`           |512 KB    |256 KB (512 KB on PPC) <br>![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB   |256 KB <br>![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB          |256 KB <br>![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB          |1 MB      |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
```

Comment:
The default only changed to 512KB on xlinux for jdk17+, and was already 512KB on plinux, zlinux still uses 256K no matter the version.